ga4gh/backends.py

```diff
@@ -186,19 +186,19 @@ def searchVariants(self, request):
         else:
             readCols = self._table.columns()[:self._firstSamplePosition]
             callSetIds = request.callSetIds
-            for csid in request.callSetIds:
-                cols = [c for name, c in self._sampleCols[csid]]
+            for callSetId in request.callSetIds:
+                cols = [col for name, col in self._sampleCols[callSetId]]
                 readCols.extend(cols)
         # Now we get the row positions for the sample columns
```

Comment:
Sounds good on both points, but I'd like to push this off to a follow up PR that addresses code issue in the backends class in general.